<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWindow</class>
 <widget class="QMainWindow" name="MyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>610</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>655</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Rysowanie</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="rysujFrame">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyjście</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cleanButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Czyść</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>261</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Rysowanie</string>
     </attribute>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>112</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Rysuj Linie</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>112</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Rysuj Elipse</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>48</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>3</number>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>112</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Rysuj Koło</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>112</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Zamaluj</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Rysuj wielokąt</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Kolory</string>
     </attribute>
     <widget class="QSlider" name="verticalSlider_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>16</width>
        <height>141</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>160</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> H      S     V</string>
      </property>
     </widget>
     <widget class="QSlider" name="verticalSlider_5">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>16</width>
        <height>141</height>
       </rect>
      </property>
      <property name="maximum">
       <number>99</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QSlider" name="verticalSlider_4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>16</width>
        <height>141</height>
       </rect>
      </property>
      <property name="maximum">
       <number>359</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QSlider" name="verticalSlider_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>16</width>
        <height>141</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> R      G     B</string>
      </property>
     </widget>
     <widget class="QSlider" name="verticalSlider">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>16</width>
        <height>141</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QSlider" name="verticalSlider_6">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>16</width>
        <height>141</height>
       </rect>
      </property>
      <property name="maximum">
       <number>99</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Warstwy</string>
     </attribute>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>92</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Warstwa 5</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Tryb</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Krycie</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Krycie</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>92</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Warstwa 4</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Tryb</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Krycie</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>92</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Warstwa 3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Tryb</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Krycie</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>92</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Warstwa 2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>250</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Tryb</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Krycie</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>92</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Warstwa 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>330</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Tryb</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Warstwa_5">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Normal mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Multiply mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Screen mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Overlay mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Darken mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lighten mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Difference mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Additive mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Substraction mode</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_Warstwa_6">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Normal mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Multiply mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Screen mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Overlay mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Darken mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lighten mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Difference mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Additive mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Substraction mode</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_Warstwa_7">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Normal mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Multiply mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Screen mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Overlay mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Darken mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lighten mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Difference mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Additive mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Substraction mode</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_Warstwa_8">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>250</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Normal mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Multiply mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Screen mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Overlay mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Darken mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lighten mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Difference mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Additive mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Substraction mode</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_Warstwa_9">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>330</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Normal mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Multiply mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Screen mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Overlay mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Darken mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lighten mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Difference mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Additive mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Substraction mode</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
